database/src/main/resources/db/migration/V1.57.0__NOTIF-603_anemic_tenants.sql

```diff
@@ -0,0 +1,26 @@
+-- Table: behavior_group
+DROP INDEX ix_behavior_group_account_id;
+
+-- Table: email_aggregation
+ALTER TABLE email_aggregation
+    ALTER COLUMN account_id DROP NOT NULL,
+    ALTER COLUMN org_id SET NOT NULL;
```

Comment:
This line depends on https://github.com/RedHatInsights/notifications-backend/pull/1425.